guide/payments/transactions.md

```diff
@@ -54,26 +103,26 @@ Let's lay out the entire payment process.
    layout = "full-width"
 %}
 
-#### 1. Get recipient address
+### 1. Get recipient address
 The sender needs a valid address for the payment to be sent to. This can be shared by the recipient as a QR code, in plain text, or as a payment link.
```

Comment:
Could also be NFC, same as the options for Lightning.

Reply:
Updated